src/analyzer/git-caller.server.ts

```diff
@@ -356,8 +370,9 @@ export class GitCaller {
 
   async getCommitCount() {
     if (!this.branch) throw Error("Branch is undefined")
-    const result = await runProcess(this.repo, "git", ["rev-list", "--count", this.branch])
-    return result as number
+    const args = ["log", this.branch, '--format="%at"']
+    const result = await runProcess(this.repo, "git", args) as string
+    return result.split(/\s/).length
```

Comment:
Why was this changed? Is it faster than the old method?

Reply:
I think at one point I wanted to be able to get the commit count of a file, which rev-list does not allow, but I no longer need to get the commit count for a file, so this should be reverted